libnetdata/socket/socket.c

```diff
@@ -1010,6 +1010,44 @@ int accept_socket(int fd, int flags, char *client_ip, size_t ipsize, char *clien
             strncpyz(client_ip, "UNKNOWN", ipsize - 1);
             strncpyz(client_port, "UNKNOWN", portsize - 1);
         }
+        char client_host[NI_MAXHOST+1];
+        if(getnameinfo((struct sockaddr *)&sadr, addrlen, client_host, (socklen_t)ipsize, NULL, 0, NI_NAMEREQD | NI_NUMERICSERV) != 0)
+            client_host[0] = 0;
+        else {
```

Comment:
We're in the process of establishing a code style convention for C code. Please consult CONTRIBUTING.md for our style guide.

Reply:
It seems to match the style guide (four space indent, not using braces around a single line body). Could you tell me how it is in the wrong style?

Reply:
Well, for starters the 120 char line limit is broken in several places (which is something you could argue about the rest of the file too, but we only started observing the style guide last week, and we'll have to deal with the monstrous reformatting PR at some point soon)

Reply:
Oh yikes that is 160 characters long, I'll fix that...

Reply:
I think it now matches the kernel style guide + exceptions.